modules/ROOT/pages/manage-api-health-checks.adoc

```diff
@@ -0,0 +1,37 @@
+= Managing API Health Checks
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API health checks enable you to check connection attempts to a specific endpoint. When you configure the health check, specify a specific status code that indicates a healthy connection to the endpoint. If the endpoint connection attempt gives a status that violates the set parameters, an email address you specify receives an email noting the failed health check.
+
+You can use any 1xx, 2xx, 3xx, 4xx, or 5xx status codes for your health checks. The default health check states that the status code must equal 200, which is the OK status. If you expect successful connections to your endpoints, 2xx status codes are ideal connection results.
+
+For example, when the American Flights API is live, the status code should return 200, indicating the API can receive connections. When you configure a health check with these parameters, you receive an email when the endpoint has a status other than 200.
+
```

Comment:
@sarathecoubian following are the topics suggested to add in the docs:
1. Health check schedule: It will be run every 15 minute. 
2. Health check states: Has 3 states, Add, passed and failed 
3. Execution summary: Customer can learn more about the run in this section. Demo avail here: https://drive.google.com/drive/search?q=demos
4. Test now feature: Customer can test the health check before applying it. Same feature as AFM. Demo avail here: https://drive.google.com/drive/search?q=demos

Reply:
https://drive.google.com/drive/search?q=demos here please refer to "sprint demo nov 17" video for execution summary and test now feature.